src/components/List/List.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { ListProps } from '../../types'
+import { SubHeader, Wrapper, ListItems } from './List.styles'
```

Comment:
```suggestion
import * as Styled from './List.styles'
```

이런 식으로 import 하면 JSX 마크업만 보고 '이건 Styled 컴포넌트구나' 알기 쉬운 것 같아요.

Reply:
넵 저렇게 수정 하도록 하겠습니다